app/assets/javascripts/modules/track-briefing.js

```diff
@@ -0,0 +1,34 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function TrackBriefing () {}
```

Comment:
Could we generalise this function name as there's nothing specific here to the briefings page? 
```suggestion
  function TrackLinks () {}
```

Reply:
TrackLinks conflicts with an existing module I believe called TrackLinks - I can call it `TrackContentLinks` or something similar?

Reply:
Ah do you have a link to the module? I can't seem to find it. Also I wonder if that module has the functionality we need?

Reply:
Sorry, it's called TrackClick:
https://github.com/alphagov/static/blob/master/app/assets/javascripts/modules/track-click.js

Reply:
Ah thank you! - so I think TrackLink might be better just because realistically you could use this anywhere on the page - not just content